source/client/renderer/PatchManager.cpp

```diff
@@ -31,13 +31,27 @@ void PatchManager::LoadPatchData(const std::string& patchData)
 	while (std::getline(patchDataStream, currLine))
 	{
 		if (currLine.empty()) continue;
+        if (currLine.at(currLine.size() - 1) == '\r')
+        {
+            // Ignore Windows line-endings when processing file on Unix systems
+            // How would we possibly standardize this if the game isn't the thing writing the file?
+            currLine = currLine.substr(0, currLine.size() - 1);
+        }
+		if (currLine.empty()) continue;
```

Comment:
Also indentation inconsistency, maybe I won't make more comments about it
Change itself is maybe useful but maybe Mac OS 9 line endings can be also handled

Reply:
BTW, you can just `.resize(.size() - 1)`